### PR TITLE
Install ember cli mocha

### PR DESCRIPTION
**Pull request by flexyford**
Chose Ember-cli-mocha for our tests. Wrote an initial test which makes sure the component renders.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,15 +1,19 @@
 {
   "name": "virtual-each",
   "dependencies": {
-    "ember": "1.13.7",
+    "ember": "components/ember#release",
```
Comment (Robdel12) [inline bower.json]: Any reason for this change? I think we'd like to keep the min ember dep at 1.13.x

Comment (flexyford) [inline bower.json]: ya ur right, this was unintentional, thanks

**Comment by jasonmit** [comment]
I'm fine with mocha :+1:

File: bower.json [hunk 2]
```diff
@@ -3,13 +3,17 @@
...
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
+    "ember-cli-test-loader": "ember-cli/ember-cli-test-loader#0.1.3",
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
     "ember-qunit": "0.4.9",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.18",
     "jquery": "^1.11.3",
     "loader.js": "ember-cli/loader.js#3.2.1",
-    "qunit": "~1.18.0"
+    "qunit": "~1.18.0",
```
Comment (jasonmit) [inline bower.json]: Can be removed

File: bower.json [hunk 3]
```diff
@@ -3,13 +3,17 @@
   "dependencies": {
     "ember": "1.13.7",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
+    "ember-cli-test-loader": "ember-cli/ember-cli-test-loader#0.1.3",
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
     "ember-qunit": "0.4.9",
```
Comment (jasonmit) [inline bower.json]: Can be removed

File: bower.json [hunk 4]
```diff
@@ -3,13 +3,17 @@
...
     "ember-qunit-notifications": "0.0.7",
```
Comment (jasonmit) [inline bower.json]: Can be removed

**Pull request merged**